index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="description" content="03/23/2023 - This is a Catholic Music Concert to be ready for 2023 in Rome city of Italy in July month.">
+  <meta name="keyword" content="Catholic, Music, Meet, Cristian">
+  <title>Catholic Music Concert - United by the same Faith</title>
+  <link href="https://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/3848942bcd.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>  
+  <header>
+    <div class="mylinks">
+      <ul class="link-menu">
+        <li class="link-item"><a ><i class="fa-brands fa-facebook-f"></i></a></li>
+        <li class="link-item"><a ><i class="fa-brands fa-twitter"></i></a></li>
+        <li class="link-item"><a >English</a></li>
+        <li class="link-item"><a >My Page</a></li>
+        <li class="link-item"><a >Logout</a></li>
+      </ul>
+    </div>
+    <div class="menu-cont">
+      <i class="fa-sharp fa-solid fa-bars"></i>
+      <a href="index.html"><img src="images/logo.png" alt="Logo Concert Music Catholic" class="logo"></a>
+      <nav class="nav-menu">
+        <ul class="menu-dk">
+          <li><a class="menu-link" href="about.html">About</a></li>
+          <li><a class="menu-link" href="#program">Program</a></li>
+          <li><a class="menu-link" href="#join">Join</a></li>
+          <li><a class="menu-link" href="#sponsors">Sponsor</a></li>
+          <li><a class="menu-link" href="#news">News</a></li>
+        </ul>
+        <button class="btn-campain" type="button"><a href="#">CMC World</a></button>
+      </nav>
+    </div>      
+  </header>  
+  <div class="page">
+    <main>
+      <section class="principal">
+      <div class="tittle">
+        <h2 class="txt-tittle">"United by the same Faith"</h2>
+        <h1 >CATHOLIC MUSIC CONCERT<br>WORLD TOUR <br>July 2023</h1>
+        <p class="description">In july, a series of presentations will be held in Rome, Italy, to praise and give glory to our Lord Jesus Christ.</p>
+        <div class="date-site">
+          <p class="date">2023.07.20(THU) ~ 22(SAT)</p>
+          <p class="site">@ Auditorio Parco della Musica <br> Roma RM, Italia</p>
+        </div>        
+      </div>
+      </section>
+      <section class="main-program">
+        <div class="main-container">
```

Comment:
- [ ] The navbars and all the highlighted sections of the home page are missing, they are not displayed on the desktop version.
- [ ] On the mobile version the header section is missing as well and the hamburger button is not well aligned, kindly fix the margins to match the design.

Reply:
Hello @algerina, I am surprised by the observations you make since when I see my work I see it complete and aligned, I'll give you some test photos.
I asked my code partner to check if he could see it and it seems he sees it correctly too.
I would ask you to send me some screenshots to know exactly what I should correct, thank you very much.

![image](https://user-images.githubusercontent.com/110146573/227601478-2f3602c0-8c86-46ef-863a-1ed82aa65832.png)

![image](https://user-images.githubusercontent.com/110146573/227602067-312669ce-abc3-47b9-9cae-8e09b024218f.png)